State.lua

```diff
@@ -1066,6 +1066,27 @@ local function removeBuff( aura )
 end
 state.removeBuff = removeBuff
 
+-- Extend buff function, defaults to 1 second if no value supplied, breaks out if supplied a 0
+local function extendBuff( aura, duration)
```

Comment:
*duration* seems like the wrong name. Maybe *delta* instead? 

Also missing a space before the close parenthesis.

Reply:
I just made it the same as other functions which all take in duration, probably up to @Hekili on that on?